data/tmpl/values/content1.html

```diff
@@ -59,6 +61,7 @@
         <h2 class="token-awarded hidden" aria-hidden="true">Here’s your badge!</h2>
       </footer>
     </section>
+    <p class="step hidden">1</p>
```

Comment:
Seems not, as you see you have it on others as well. What is the utility of these items? Can we instead use a `data` attribute on an already existing element?